rskj-core/src/main/java/co/rsk/logfilter/BlocksBloomStore.java

```diff
@@ -29,9 +31,11 @@
  * Created by ajlopez on 05/02/2019.
  */
 public class BlocksBloomStore {
+    private static final Logger logger = LoggerFactory.getLogger("blooms");
+
     private final int noBlocks;
     private final int noConfirmations;
-    private final Map<Long, BlocksBloom> blocksBloom = new ConcurrentHashMap<>();
+    private final Map<Long, BlocksBloom> blocksBloomCache = new ConcurrentHashMap<>();
```

Comment:
I guess here we could use `HashMap` instead of `ConcurrentHashMap`. Access to this filed is synchronized in this class.

Reply:
Done!